### DISCUSSION TITLE
archive/2310-jp-travel/

### DISCUSSION THREAD

**Discussion by giscus** [jerrylususu/jerrylususu.github.io#32] [Blog Comments]
# archive/2310-jp-travel/

为什么要写下来？因为不写就真的忘记了。 注：部分不适宜面向所有人公开的内容已用 [REDACTED] 自主规制。 day 0 香港-东京 永东巴士去hk，原来以为是真正的大巴，

https://nekonull.me/archive/2310-jp-travel/

**Comment by gruexx** [comment]
是我理想中的一个人的旅行啊🤔
